packager/README.md

```diff
@@ -0,0 +1,51 @@
+## Assumptions
```

Comment:
Some of these assumptions (e.g. ampbyexample references) are out of date, perhaps update/delete/flag somehow to avoid future confusion.